src/Compilers/CSharp/Test/Syntax/Parsing/ExpressionParsingTests.cs

```diff
@@ -3540,8 +3540,7 @@ public void TestTargetTypedDefault()
             var expr = this.ParseExpression(text, TestOptions.ExperimentalParseOptions);
```

Comment:
>ExperimentalParseOptions [](start = 62, length = 24)

This is the only test in this file that is using ExperimentalParseOptions. Consider changing it to use CSharp7 and have have a test with 7.1 next to other ```default``` tests.